components/widget/ratings/ratings_list.lua

```diff
@@ -216,7 +216,7 @@ function RatingsList:render()
 			classes = { 'ranking-table__header-row' },
 		},
 		Array.flatMap(teamRows, FnUtil.identity),
-		tableFooter
+		addNavigationButton and tableFooter
```

Comment:
```suggestion
		addNavigationButton and tableFooter or nil
```
we might need this, else it likely evaluates as `false` and tries to pass a bool to stuff that can't handle bool